test/test-app.js

```diff
@@ -18,11 +18,9 @@ function exec(command, args) {
   return execResult.stdout.toString();
 }
 
-function setupGit(dir, done) {
+function addFile(dir, done) {
   var fileName = path.join(dir, 'index.html');
   fs.writeFile(fileName, '', done);
-  exec('git', 'config user.email "you@example.com"');
-  exec('git', 'config user.name "Your Name"');
```

Comment:
This code is not efficient for Travis. git in Travis requires to setup user settings globally. And we can`t apply global settings in a test case. These tests work fine in developers environment and fail in Travis. So we should store these settings in `travis.yml`.